### PR TITLE
Activation ops

### PR DESCRIPTION
**Pull request by ZihengJiang**

### CODE REVIEW

**Comment by tqchen** [comment]
should they be moved to nn?

**Comment by ZihengJiang** [comment]
@tqchen you're right

**Comment by piiswrong** [comment]
these are more like unary I think

**Comment by tqchen** [comment]
depending on the syntax, usually math functions should goto unary, like tanh, while relu is quite like a nn function. But i think either way is fine

**Comment by piiswrong** [comment]
I thought we had sigmoid already

File: src/operator/sigmoid-inl.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                 const std::vector<TBlob>& inputs,
+                 const std::vector<OpReqType>& req,
+                 const std::vector<TBlob>& outputs) {
+  using namespace mshadow;
+  using namespace mxnet_op;
+  Stream<xpu> *s = ctx.get_stream<xpu>();
+
+  CHECK_EQ(inputs.size(), 1U);
+  CHECK_EQ(outputs.size(), 1U);
+  MSHADOW_TYPE_SWITCH(outputs[0].type_flag_, DType, {
```
Comment (piiswrong) [inline src/operator/sigmoid-inl.h]: this is duplicating code. You can replace UnaryCompute with Kernel Launch and move these functions to unary.

**Pull request merged**